### PR TITLE
[WebNN EP] Update Prelu restriction for CPU backend

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@fdwr, @guschmue, @fs-eire, PTAL, thanks!

**Review by guschmue — DISMISSED** [review]

**Comment by Honry** [comment]
@guschmue, could you please retrigger the failure CIs, thanks!

**Comment by Honry** [comment]
@fdwr, @guschmue, I just solve the doc conflict, please help retrigger the CIs, thanks!

File: js/web/docs/webnn-operators.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ operators and the supported opset domain/versions in **WebNN EP** by ONNX Runtim
 | Not | ai.onnx(7+) | logicalnot | ✓ | ✓ | |
 | Pad | ai.onnx(7-10, 11-12, 13-17, 18, 19-20, 21+) | pad | ✓ | ✓ | modes == 'wrap' is not supported |
 | Pow | ai.onnx(7-11, 12, 13-14, 15+) | pow | ✓ | ✓ | |
-| PRelu | ai.onnx(7-8, 9-15, 16+) | prelu | ✓ | ✓ | WebNN CPU restricts slope to be a static value |
+| PRelu | ai.onnx(7-8, 9-15, 16+) | prelu | ✓ | ✓ | WebNN CPU backend restricts the last dimension of input and slope to be same (Chromium issue: https://issues.chromium.org/issues/335517470) |
```
Comment (fdwr) [inline js/web/docs/webnn-operators.md:62]: Such a randomly arbitrary restriction 🫡.

**Review by fdwr — APPROVED** [review]

**Pull request merged**